src/handlers/auth_handlers.rs

```diff
@@ -28,6 +29,13 @@ use crate::{
     },
 };
 
+use hmac::{Hmac, Mac};
+use sha2::Sha256;
+use rand::Rng;
+use base64::{Engine as _, engine::general_purpose};
+
+const HMAC_SECRET: &[u8] = b"your-secret-key-here";
```

Comment:
Store secrets in `src/utils/constants` and then import them here.

Reply:
Oh sorry for blunder